plugins/destination/kafka/client/write.go

```diff
@@ -82,3 +71,7 @@ func (c *Client) Write(ctx context.Context, tables schema.Tables, res <-chan *de
 
 	return nil
 }
+
+func (*Client) WriteTableBatch(ctx context.Context, table *schema.Table, data [][]any) error {
```

Comment:
maybe just embed [destination.UnimplementedManagedWriter](https://pkg.go.dev/github.com/cloudquery/plugin-sdk@v1.27.0/plugins/destination#UnimplementedManagedWriter)?

Reply:
It already is:
https://github.com/cloudquery/cloudquery/blob/24b0219ca14628da3f3188efb463915064bf9a0d/plugins/destination/kafka/client/client.go#L18-L33

Reply:
then explicit func isn't required